cluster-proxy/src/main/java/com/sequenceiq/cloudbreak/clusterproxy/ConfigRegistrationRequest.java

```diff
@@ -22,6 +28,12 @@
     @JsonProperty
     private List<String> certificates;
 
+    @JsonProperty
+    private boolean useTunnel;
```

Comment:
I think this should be an enum instead of boolean

Reply:
This would mean a contract change for cluster proxy. And we currently only have either ccm enabled or direct communication. Could we please revisit this later?

Reply:
This is a flag to ask cluster-proxy to enable/disable ccm for a given workload. This needs to be boolean.